pilot/pkg/security/authn/policy_applier.go

```diff
@@ -42,6 +42,10 @@ type PolicyApplier interface {
 	// PortLevelSetting returns port level mTLS settings.
 	PortLevelSetting() map[uint32]*v1beta1.PeerAuthentication_MutualTLS
 
+	MtlsChecker
+}
+
+type MtlsChecker interface {
```

Comment:
MtlsChecker is confusing with another MtlsChecker in xds and also it is not actually checking any thing., May be call it `MtlsPolicy`?

Reply:
sounds good